libs/components/forms/src/lib/modules/selection-box/selection-box.component.html

```diff
@@ -50,4 +47,4 @@
     <ng-content select="sky-radio"> </ng-content>
     <ng-content select="sky-checkbox"> </ng-content>
   </div>
-</div>
+</label>
```

Comment:
☝️ This is the principal change. This will allow the user to click on the wrapper to activate the child checkbox/radio without failing the `nested-interactive` Axe rule.